hs_core/hydroshare/resource.py

```diff
@@ -1032,15 +1033,22 @@ def submit_resource_for_review(request, pk):
                               "metadata or content files, or this resource contains referenced "
                               "content, or this resource type is not allowed for publication.")
 
-    resource.raccess.review_pending = True
-    resource.raccess.immutable = True
-    resource.raccess.save()
-
-    # we assume that there is a user associated with the "default_from_email"
-    user_to = User.objects.get(email__iexact=settings.DEFAULT_FROM_EMAIL)
+    try:
+        user_to = User.objects.get(email__iexact=settings.DEFAULT_FROM_EMAIL)
+    except User.DoesNotExist:
+        user_to = create_account(
+            email=settings.DEFAULT_FROM_EMAIL,
```

Comment:
Are we assuming` settings.DEFAULT_FROM_EMAIL` will have a value? Otherwise, account creation will fail.

Reply:
Good point. I think we make this assumption throughout HS, so probably ok?